### PR TITLE
privval: missing privval type check in SetPrivValidator

### PR DESCRIPTION
**Pull request by JayT106**
fixes https://github.com/tendermint/tendermint/pull/6578#discussion_r662180308

ran with `./build/runner -f networks/ci.toml` and checked the docker log message to confirm no error message

### CODE REVIEW

File: types/priv_validator.go [hunk 1]
```diff
@@ -17,9 +17,10 @@ type PrivValidatorType uint8
 const (
 	MockSignerClient      = PrivValidatorType(0x00) // mock singer
 	FileSignerClient      = PrivValidatorType(0x01) // singer client via file
-	RetrySignerClient     = PrivValidatorType(0x02) // singer client via socket
-	SignerClient          = PrivValidatorType(0x03) // singer client via gRPC
+	RetrySignerClient     = PrivValidatorType(0x02) // singer client with retry via socket
+	SignerSocketClient    = PrivValidatorType(0x03) // singer client via socket
 	ErrorMockSignerClient = PrivValidatorType(0x04) // error mock singer
+	SignergRPCClient      = PrivValidatorType(0x05) // singer client via gRPC
```
Comment (cmwaters) [inline types/priv_validator.go:23]:
```suggestion
	RetrySignerClient     = PrivValidatorType(0x02) // signer client with retry via socket
	SignerSocketClient    = PrivValidatorType(0x03) // signer client via socket
	ErrorMockSignerClient = PrivValidatorType(0x04) // error mock singer
	SignergRPCClient      = PrivValidatorType(0x05) // signer client via gRPC
```

Comment (cmwaters) [reply types/priv_validator.go:23]: Also do we want the g in gRPC to be capitalized else I feel like it's not so recognizable (looks like an RPCClient)

File: types/priv_validator.go [hunk 2]
```diff
@@ -17,9 +17,10 @@ type PrivValidatorType uint8
 const (
 	MockSignerClient      = PrivValidatorType(0x00) // mock singer
 	FileSignerClient      = PrivValidatorType(0x01) // singer client via file
```
Comment (cmwaters) [inline types/priv_validator.go:19]:
```suggestion
	FileSignerClient      = PrivValidatorType(0x01) // signer client via file
```

**Review by cmwaters — APPROVED** [review]
Just some minor typos.

I ran the previously failing e2e test and this seemed to have fixed it. Thanks!

**Comment by JayT106** [comment]
@cmwaters, Thanks for the review. Updated the fixes.

**Pull request merged**